### DISCUSSION TITLE
Is there a way to disable the retainer flavor text when they come back from a venture?

### DISCUSSION THREAD

**Discussion by elrandir** [karashiiro/TextToTalk#213] [Q&A] [answered]
I tried exclusions with their names but that didn't do the trick, so far I just have been disabling the addon with a hotkey when I collect the ventures. Is there another way? Thank you.

**Comment by johnysandels** [comment]
You'll probably have to regex or filter the entire sentence

**Reply by elrandir** [reply]
Could you please give an example? I tried adding the entire line including the npc name: before it, but that didn't work. I'm not familiar with regex and google is not helping me find what I need, I tried adding ^ in front of the npc name but that didn't help. Thank you in advance!

**Reply by johnysandels** [reply, marked as answer]
I've just tested it and it actually don't seem to exclude text from the dialogue box. only text from the chat log. I'm going to open an issue for this. #214
